File: MyApplication21/app/src/main/java/com/parkch/myapplication/MainActivity.java
package com.parkch.myapplication;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    Button james, sophia;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        james = (Button)findViewById(R.id.back2);
        james.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(getApplicationContext(), james.class);
                startActivity(intent);
            }
        });

        sophia = (Button)findViewById(R.id.button2);
        sophia.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(getApplicationContext(), sophia.class);
                startActivity(intent);
            }
        });
    }
}

File: Kotlin Start/app/src/main/java/com/example/kotlinstart/MainActivity.kt
package com.example.kotlinstart

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        button1.setOnClickListener() {
            startActivity(Intent(this,BmiJavaActivity::class.java))
        }

        button2.setOnClickListener() {
            startActivity(Intent(this,BmiKotlinActivity::class.java))
        }
    }
}
